index.js

```diff
@@ -22,14 +21,22 @@ const exec = (cmd, args) => {
 	).filter(Boolean);
 };
 
+const install = (bin, useYarn) => () => execa(bin, ['install'])
+	.then(c => c, e => {
+		if (e.code === 'ENOENT' && useYarn) {
+			return execa('npm', ['install']);
+		}
+		throw e;
+	});
+
 module.exports = (input, opts) => {
 	input = input || 'patch';
 
 	opts = Object.assign({
 		cleanup: true,
 		publish: true
 	}, opts, {
-		yarn: opts && opts.yarn !== false && hasYarn()
```

Comment:
I'm not checking for `yarn` anymore since we are defaulting to it, and falling back when is not found.